### DISCUSSION TITLE
How to make foreground bold?

### DISCUSSION THREAD

**Discussion by cofeek-codes** [rose-pine/neovim#310] [Q&A] [answered]
this is my config
```lua
return {

  {
    "rose-pine/neovim",
    name = "rose-pine",
    opts = {
      variant = "main",
      styles = {
        transparency = true,
        italic = false,
        bold = false,
      },

      highlight_groups = {
        CursorLineNr = { link = "LineNr" },
        MatchParen = { fg = "#f6c177", bg = "#908caa" },
        CursorLine = { bg = "none" },
      },
    },
  },
}

```
i want to make MatchParen group to have no background and foreground to be bold. i could not figure  out how to do so. Please help

**Comment by cofeek-codes** [comment]
ok i managed to make it bold, but i still need to make background transparent. `bg = none` just makes in black
`MatchParen = { fg = "#f6c177", bg = "none", bold = true },`

**Reply by mvllow** [reply, marked as answer]
There's a default blend applied to "MatchParen" by our theme so you'll want to add `inherit = false` to your highlight override

**Reply by cofeek-codes** [reply]
It works, thank you!
